setup/test_artifacts.yml

```diff
@@ -0,0 +1,39 @@
+parameters:
+  - name: go_version
+    type: string
+  - name: artifact_name
+    type: string
+  - name: download_path
+    type: string
+  - name: item_pattern
+    type: string
+
+steps:
+  #TODO: Change the org name after merging to main
+  - script: |
+      git clone https://github.com/dphulkar-msft/azure-storage-azcopy
```

Comment:
Why clone the repository like this when we can just do checkout:self in the pipeline?

Reply:
removed as checkout:self takes care of it.